recipes/sobjectizer/all/conanfile.py

```diff
@@ -1,5 +1,9 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import get, copy, rmdir, collect_libs
+from conan.tools.scm import Version
 import os
 
 required_conan_version = ">=1.43.0"
```

Comment:
```suggestion
required_conan_version = ">=1.53.0"
```
For `self.options.rm_safe()`.